source/go-perun/app_tutorial/client.rst

```diff
@@ -0,0 +1,148 @@
+.. _app_client:
+
+Client
+======
+In this section, we describe the app channel client that is used for opening a channel with the Tic Tac Toe app instantiated.
+Much of the app channel client is similar to the :ref:`payment channel client <payment_client>`, and we will reuse some of the code here. The code will be placed in package ``client``.
+
+.. code-block:: go
+
+    package client
+
+Constructor
+-----------
+The main part of our payment channel client is placed in ``client/client.go``.
+Our client is of type ``AppClient``, which is similar to the ``PaymentClient``.
+The only additional parameters here are ``stake`` and ``app``.
+
+.. literalinclude:: ../../perun-examples/app-channel/client/client.go
+    :emphasize-lines: 6,7
+    :caption: `👇 This code on GitHub. <https://github.com/perun-network/perun-examples/blob/689b8cdfef8ef8fb527723d52e6ce36dfe1b661c/app-channel/client/client.go#L41>`__
```

Comment:
For captions two underscores are required. Otherwise:
` WARNING: Invalid caption: /Users/lr/-/source/go-perun/app_tutorial/client.rst:4: (WARNING/2) Duplicate explicit target name: "👇 this code on github.".
`